### PR TITLE
Remove buttons above user and group search

### PR DESCRIPTION
**Pull request by cconard96**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

To avoid double scrollbar on these search pages, the buttons that were above the search have been moved to the top of the page. This also helps keep the UI consistent with other pages.
Even after moving the buttons, there was still a double scrollbar but without any clear reason so I made a small adjustment to the content-margin CSS variable to compensate.

There are two other pages with display alerts/messages above the search which also give the double scrollbar issue, but the fix is more complex so it will be done in another PR.

### CODE REVIEW

File: src/User.php [hunk 1]
```diff
@@ -130,6 +130,20 @@ public static function getAdditionalMenuOptions()
...
+    public static function getAdditionalMenuLinks()
+    {
+        $links = [];
+        if (Auth::useAuthExt() && Session::haveRight('user', self::IMPORTEXTAUTHUSERS)) {
+            if (static::canCreate()) {
+                $ext_auth_label = __s('Add from an external source');
+                $links['<i class="ti ti-user-cog"></i>' . $ext_auth_label] = 'user.form.php?new=1&amp;ext_auth=1';
+            }
+            if (static::canCreate() || static::canUpdate()) {
+                $links['<i class="ti ti-settings me-2"></i>' . __s('LDAP directory link')] = "ldap.php";
```
Comment (orthagh) [inline src/User.php:142]:
```suggestion
                $links['<i class="ti ti-user-cog"></i><span>' . $ext_auth_label . '</span>'] = 'user.form.php?new=1&amp;ext_auth=1';
            }
            if (static::canCreate() || static::canUpdate()) {
                $links['<i class="ti ti-settings"></i><span>' . __s('LDAP directory link') . '</span>'] = "ldap.php";
```

this to make auto spacing with the icon.
I didn't make all other iterations, but this need to be applied on all getAdditionalMenuLinks

File: src/User.php [hunk 2]
```diff
@@ -130,6 +130,20 @@ public static function getAdditionalMenuOptions()
         return false;
     }
 
+    public static function getAdditionalMenuLinks()
+    {
+        $links = [];
+        if (Auth::useAuthExt() && Session::haveRight('user', self::IMPORTEXTAUTHUSERS)) {
+            if (static::canCreate()) {
+                $ext_auth_label = __s('Add from an external source');
+                $links['<i class="ti ti-user-cog"></i>' . $ext_auth_label] = 'user.form.php?new=1&amp;ext_auth=1';
```
Comment (orthagh) [inline src/User.php:139]: Missed one

```suggestion
                $links['<i class="ti ti-user-cog"></i><span>' . $ext_auth_label . '</span>'] = 'user.form.php?new=1&amp;ext_auth=1';
```

**Review by orthagh — APPROVED** [review]

File: src/Group.php [hunk 3]
```diff
@@ -227,31 +227,17 @@ public function showForm($ID, array $options = [])
         ]);
     }
 
-    /**
-     * Print a good title for group pages
-     *
-     *@return void
-     **/
-    public function title()
```
Comment (cedric-anne) [inline src/Group.php:235]: Please add a changelog entry.

File: src/User.php [hunk 4]
```diff
@@ -2490,49 +2504,6 @@ public function blankPassword()
         }
     }
 
-
-    /**
-     * Print a good title for user pages.
-     *
-     * @return void
-     */
-    public function title()
```
Comment (cedric-anne) [inline src/User.php:2499]: Please add a changelog entry.

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**